lib/openscap_parser/test_result.rb

```diff
@@ -4,10 +4,41 @@ module OpenscapParser
   module TestResult
     def self.included(base)
       base.class_eval do
-        private
+        def score
+          @score ||= test_result_node &&
+            test_result_node.search('score').text.to_f
+        end
+
+        def start_time
+          @start_time ||= test_result_node &&
+            DateTime.parse(test_result_node['start-time'])
+        end
+
+        def end_time
+          @end_time ||= test_result_node &&
+            DateTime.parse(test_result_node['end-time'])
+        end
+
+        def test_result_profiles
```

Comment:
@bastilian funnily enough, `MiniTest::Test` only has a `run` method, but it looks for methods which start with `test_`. That creates the scenario you predicted would happen! :stuck_out_tongue_closed_eyes: So I've had to wrap the `profiles_test` tests in a dummy test class like you suggested.